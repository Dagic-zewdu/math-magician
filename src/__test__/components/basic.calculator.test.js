import React from 'react';
import renderer from 'react-test-renderer';
import Calculator from '../../components/calculator/basic.calculator';

describe('should render without problems', () => {
  it('should render the calculator', () => {
    const tree = renderer.create(<Calculator />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
