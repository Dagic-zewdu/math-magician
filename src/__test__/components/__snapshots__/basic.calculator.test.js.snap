// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render without problems should render the calculator 1`] = `
<div
  className="container"
>
  <div
    className="d-grid gray"
  >
    <div
      className="g-item gray"
      style={
        Object {
          "fontSize": "4rem",
          "gridColumn": "1/5",
          "justifyContent": "flex-end",
        }
      }
    >
      0
    </div>
    <button
      className="g-item white"
      onClick={[Function]}
      type="button"
    >
      AC
    </button>
    <button
      className="g-item white"
      onClick={[Function]}
      type="button"
    >
      +/-
    </button>
    <button
      className="g-item white"
      onClick={[Function]}
      type="button"
    >
      %
    </button>
    <button
      className="g-item orange"
      onClick={[Function]}
      type="button"
    >
      รท
    </button>
    <button
      className="g-item white"
      onClick={[Function]}
      type="button"
    >
      7
    </button>
    <button
      className="g-item white"
      onClick={[Function]}
      type="button"
    >
      8
    </button>
    <button
      className="g-item white"
      onClick={[Function]}
      type="button"
    >
      9
    </button>
    <button
      className="g-item orange"
      onClick={[Function]}
      type="button"
    >
      x
    </button>
    <button
      className="g-item white"
      onClick={[Function]}
      type="button"
    >
      4
    </button>
    <button
      className="g-item white"
      onClick={[Function]}
      type="button"
    >
      5
    </button>
    <button
      className="g-item white"
      onClick={[Function]}
      type="button"
    >
      6
    </button>
    <button
      className="g-item orange"
      onClick={[Function]}
      type="button"
    >
      -
    </button>
    <button
      className="g-item white"
      onClick={[Function]}
      type="button"
    >
      1
    </button>
    <button
      className="g-item white"
      onClick={[Function]}
      type="button"
    >
      2
    </button>
    <button
      className="g-item white"
      onClick={[Function]}
      type="button"
    >
      3
    </button>
    <button
      className="g-item orange"
      onClick={[Function]}
      type="button"
    >
      +
    </button>
    <button
      className="g-item white"
      onClick={[Function]}
      style={
        Object {
          "gridColumn": "1/3",
        }
      }
      type="button"
    >
      0
    </button>
    <button
      className="g-item white"
      onClick={[Function]}
      type="button"
    >
      .
    </button>
    <button
      className="g-item orange"
      onClick={[Function]}
      type="button"
    >
      =
    </button>
  </div>
</div>
`;
